### DISCUSSION TITLE
Continued decoupled Orchard Core CMS with Hisham Bin Ateya - Orchard Core Pair Programming by Lombiq

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16225] [Show and tell]
Join this session of Orchard Core Pair Programming by Lombiq! In these, we do an hour of pair programming with an Orchard Core community member about a project of theirs. We learn together a lot, share best practices, and write some good code. All this is live, and you can join us with your questions! Check it out here: https://www.youtube.com/watch?v=yeznXsNF9sU. See you at 5 PM UTC on the 11th!

@hishamco, a core contributor of Orchard Core, joins us again with a follow-up to the previous stream (https://www.youtube.com/live/EC8C4tHpxnE). We'll see a decoupled CMS project he was building for Quality Training Academy for Health Training from Saudi Arabia.

Your host and the "navigator" of the pair programming session will be me.

Would you like to be our guest? Just write a comment here!

**Comment by Piedone** [comment]
We're starting the stream with Hisham now! Tune in here: https://www.youtube.com/watch?v=yeznXsNF9sU

**Comment by hishamco** [comment]
I will update the repo https://github.com/OrchardCoreContrib/OrchardCoreContrib.Samples that will contain a Decoupled CMS sample, then we can do headless and coupled with various types of technologies
